app/controllers/hearing_days_controller.rb

```diff
@@ -0,0 +1,67 @@
+class HearingDaysController < ApplicationController
+  before_action :load_and_authorize_application, :load_hearing_days, :load_hearing_details,
+                :load_hearing_events
+
+  add_breadcrumb :search_filter_breadcrumb_name, :new_search_filter_path
+  add_breadcrumb :search_breadcrumb_name, :search_breadcrumb_path
+
+  def show
+    add_breadcrumb @application.application_title
+    add_breadcrumb t(".breadcrumb", day: @hearing_day.day_string)
+  end
+
+  private
+
+  def load_and_authorize_application
+    @application = CourtDataAdaptor::Query::CourtApplication.new(params[:court_application_id]).call
```

Comment:
Should we add parameter validation to ensure parameters are present?

Reply:
A nice Rails thing helps us out here - because the params here are in the path rather than the querystring, they have to be present for the route matcher to work, so if they're missing this method wouldn't be called in the first place - it'd be handled as a 404 automatically.